<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 255);</string>
        </property>
        <property name="title">
         <string>configuration</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QComboBox" name="cbBaudrate">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="pbConnect">
           <property name="text">
            <string>connect </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbSerialportName"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pbDisConnect">
           <property name="text">
            <string>disconnect</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>baud rate:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 255);</string>
        </property>
        <property name="title">
         <string> Parameter Setting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Stop:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leStopFreq">
           <property name="text">
            <string>500000000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbSweepPoints">
           <property name="currentText">
            <string>11</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>41</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>51</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>101</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>N :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>points</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leStartFreq">
           <property name="text">
            <string>200000000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QPushButton" name="pbConfig">
           <property name="text">
            <string>Test Parameters Config</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 255);</string>
        </property>
        <property name="title">
         <string>calibrate</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="4">
          <widget class="QPushButton" name="pbThruCal">
           <property name="text">
            <string>Thru</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pbShortCal">
           <property name="text">
            <string>Short</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QPushButton" name="pbLoadCal">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pbOpenCal">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QTextBrowser" name="textNote"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="pbDoneCal">
           <property name="text">
            <string>Done Cal</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="3">
          <widget class="QPushButton" name="pbResetCal">
           <property name="text">
            <string>Reset Cal</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 255);</string>
        </property>
        <property name="title">
         <string>Testing</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="0">
          <widget class="QPushButton" name="pbSaveData">
           <property name="text">
            <string>save data to file</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="chbS11">
           <property name="text">
            <string>S11</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pbStop">
           <property name="text">
            <string>stop</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="chbS21">
           <property name="text">
            <string>S21</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pbLoadData">
           <property name="text">
            <string>load data from file</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="pbSingleStep">
           <property name="text">
            <string>single step measure</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pbRun">
           <property name="text">
            <string>run</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="pbSaveImage">
           <property name="text">
            <string>save image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="title">
       <string>Screen</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QChartView" name="chart" native="true">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 127);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1448</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
